test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py

```diff
@@ -437,8 +431,8 @@ def test_mme4(
     instance_type,
     accelerator_type,
 ):
-    if "p3" in instance_type:
-        pytest.skip("skip for p3 instance")
+    if "gpu" in image_uri:
```

Comment:
It looks like we can just use the skip_gpu pytest marker here instead of doing this skip inside the test function itself - can you do that to avoid having to add custom logic in the test?